_data/toc.yaml

```diff
@@ -1486,6 +1484,24 @@ manuals:
               title: Kubernetes driver
             - path: /build/building/drivers/remote/
               title: Remote driver
+        - path: /build/building/cache/
+          title: Optimizing builds with cache management
+        - sectiontitle: Cache backends
+          section:
+            - path: /build/building/cache-backends/
+              title: Overview
+            - path: /build/building/cache-backends/inline/
+              title: Inline
+            - path: /build/building/cache-backends/local/
+              title: Local
+            - path: /build/building/cache-backends/registry/
+              title: Registry
+            - path: /build/building/cache-backends/gha/
+              title: GitHub Actions
+            - path: /build/building/cache-backends/azblob/
+              title: Azure Blob Storage
+            - path: /build/building/cache-backends/s3/
+              title: Amazon S3
```

Comment:
I think we should put `Cache backends` under `building/cache` with `building/cache/backends` slug. WDYT?

Reply:
I like the idea of the nested slugs, but don't think we should nest them together in the menu - we'd then to write a new overview page, which wouldn't really say much I don't think.

Reply:
Hum right